runner/internal/config/config.go

```diff
@@ -120,6 +121,13 @@ func (cfg Global) String() string {
 	return string(b)
 }
 
+// Equal returns true if cfg and c are equal configurations.
+func (cfg Global) Equal(c Global) bool {
+	cfg.fieldsSet = nil
+	c.fieldsSet = nil
+	return reflect.DeepEqual(cfg, c)
+}
+
 // Override returns a new Config based on cfg with overridden values from c.
```

Comment:
I think it's the opposite now?
`// Override returns a new Config based on c with overridden values from cfg`

Reply:
It is outdated doc, this has been done on the pr dealing with extracting the cli.

This pr should be rebase on main, as only the very last commit is relevant :)